src/commands/api/DatabaseAccountTreeItemInternal.ts

```diff
@@ -57,10 +58,15 @@ export class DatabaseAccountTreeItemInternal implements DatabaseAccountTreeItem
 
     public get docDBData(): { masterKey: string; documentEndpoint: string; } | undefined {
         if (this._accountNode instanceof DocDBAccountTreeItemBase) {
-            return {
-                documentEndpoint: this._accountNode.root.endpoint,
-                masterKey: this._accountNode.root.masterKey
-            };
+            const keyCred = this._accountNode.root.credentials.filter((cred): cred is CosmosDBKeyCredential => cred.type === 'key')[0];
```

Comment:
`.at(0)`

Reply:
The 'lib' in tsconfig is too old to support .at. Changing it results in multiple build error to surface. I'll skip using .at in this PR.